wgpu-types/src/lib.rs

```diff
@@ -922,6 +933,11 @@ pub struct PrimitiveState {
     /// Setting this to something other than `Fill` requires `Features::NON_FILL_POLYGON_MODE` to be enabled.
     #[cfg_attr(any(feature = "trace", feature = "replay"), serde(default))]
     pub polygon_mode: PolygonMode,
+    /// If set to true, the primitives are rendered with conservative overestimation. I.e. any rastered pixel touched by it is filled.
+    /// Only valid for PolygonMode::Fill!
```

Comment:
is this going to be true forever? We may consider doing the `Fill { conservative: bool }` in this case.

Reply:
Thought of `Fill { conservative: bool }` as well, less surface to do the wrong thing. But that would break the api in a more uncomfortable way :/

Conservative point and line rendering is today possible on Vulkan only. Checking in Vulkan.gpuinfo.org looks at first like it's covered by most (extension support at 28.9%, the support for lines/points at 20.6% of all reports). But a closer look shows that it's basically an "is this an Nvidia card" check. https://vulkan.gpuinfo.org/displayextensionproperty.php?name=conservativePointAndLineRasterization&platform=windows